### DISCUSSION TITLE
Github Student development package- wrong location due to exchange study

### DISCUSSION THREAD

**Discussion by BetaHsu** [community/community#51930] [GitHub Education]
Hi, I'm a current student with a valid school-issued student id card. I happened to be on an exchange semester. While I tried to apply for the Education package, the site asked to access my location data and told me that they can't find my home institution according to my current location. I'd love to provide an additional document to prove my exchange status with both my home university and my exchange university indicated, but I'll need a place to upload it on the application site other than my student ID. Additionally, I could also use the email issued by my exchange university instead of the one issued by my home university, but it seems that I can't unlink the email from my GitHub account anymore. How could I potentially solve this issue? thx!!

**Comment by samus-aran** [comment]
Hi @BetaHsu 👋 

Sounds like you have multiple options:

If you want to use your home institution then you must present with documentation that explicitly state you are a remote student.

Alternatively you could add a new academic email, cancel you current GitHub Global Campus application and re-apply with your exchange university. Ideally you use your current institution that you're attending to verify your current student status. 

On top of the above the other rejection reason is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍
